ndbench-core/src/main/java/com/netflix/ndbench/core/defaultimpl/NdBenchGuiceModule.java

```diff
@@ -39,8 +40,14 @@
     protected void configure()
     {
         bind(NdBenchMonitor.class).to(FakeMonitor.class);
-        //bind(IClusterDiscovery.class).to(AWSLocalClusterDiscovery.class);
-        bind(IClusterDiscovery.class).to(LocalClusterDiscovery.class);
+        String discoveryEnv = System.getenv(NdBenchConstants.DISCOVERY_ENV);
```

Comment:
Since this issue/ comment is not a blocker, I am approving this PR, please update the instructions/ documentation whenever you can.